packages/api/src/routes/medical/__tests__/fixtures.ts

```diff
@@ -0,0 +1,67 @@
+export function makePatientData(
+  overrides: Partial<{
+    firstName: string;
+    lastName: string;
+    dob: string;
+    contact: Array<{ phone: string }>;
+    address: Array<{ state: string }>;
+  }> = {}
+) {
+  return {
+    firstName: "John",
+    lastName: "Doe",
+    dob: "1990-01-01",
+    contact: [{ phone: "555-1234" }],
+    address: [{ state: "CA" }],
+    ...overrides,
+  };
+}
+
+export function makePatient(
+  overrides: { patientData?: Partial<ReturnType<typeof makePatientData>> } & Record<
+    string,
+    unknown
+  > = {}
+) {
+  const patientData = makePatientData(overrides.patientData);
+  return {
+    id: "patient-123",
+    cxId: "cx-123",
+    data: patientData,
+    dataValues: {
+      id: "patient-123",
+      cxId: "cx-123",
+      data: patientData,
+    },
+    ...overrides,
+  };
+}
+
+export function makeEncounter(
```

Comment:
This is not returning a TcmEncounterModel, but the response of the specific pagination query that joins w/ Patient, right?

If the expectation is that it returns a TcmEncounterModel (or TcmEncounter), it'd be good to have a "Partial" input and type the output as such. The name is good, though (maybe just add `...Model` if its a model).

If it just returns that specific query, maybe we move this fixture to that folder's `__test__` folder and name this accordingly?